src/types/index.ts

```diff
@@ -173,6 +173,8 @@ export interface IStepQueryParams {
   namespace?: string;
   // e.g. 'START', 'END', 'MIDDLE'
   type?: string;
+  'previous-step'?: string;
+  'following-step'?: string;
```

Comment:
What are your thoughts about renaming these properties to the following? I think that would be easier to write a camel-cased version, there's an optional backend pull request if you want to do it: https://github.com/KaotoIO/kaoto-backend/pull/643

```suggestion
  'previousStep'?: string;
  'followingStep'?: string;
```

Reply:
fixed :+1: